data/testscommon/slash/generator.go

```diff
@@ -0,0 +1,61 @@
+package slash
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/ElrondNetwork/elrond-go-core/data"
+	"github.com/ElrondNetwork/elrond-go-core/data/block"
+	dataMock "github.com/ElrondNetwork/elrond-go-core/data/mock"
+	"github.com/ElrondNetwork/elrond-go-core/data/slash"
+	"github.com/ElrondNetwork/elrond-go-core/hashing"
+	"github.com/ElrondNetwork/elrond-go-core/hashing/blake2b"
+	"github.com/ElrondNetwork/elrond-go-core/marshal"
+	"github.com/stretchr/testify/require"
+)
+
+func GenerateSlashResults(b *testing.B, noOfPubKeys uint32, noOfHeaders uint32) map[string]slash.SlashingResult {
+	hasher, err := blake2b.NewBlake2bWithSize(16)
+	require.Nil(b, err)
+
+	headers := GenerateDistinctHeadersInfo(b, noOfHeaders, hasher)
+	slashRes := make(map[string]slash.SlashingResult, noOfPubKeys)
+	for i := 0; i < int(noOfPubKeys); i++ {
+		tmp := fmt.Sprintf("pubKey%v", i)
```

Comment:
why do we need this concat? why not directly pubKeyHash := hasher.Compute(string(pubKey))?